app/pb_kits/playbook/pb_label_value/_label_value.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.label %>
-  <%= object.value %>
+    class: object.classname) do %>
+  <%= pb_rails("caption", props: { text: object.label }) %>
```

Comment:
Doesn't this still need the conditional? Only render a label if a label is passed?

If this prop is truly required (and the conditional is inappropriate), let's make it `required: true`

Reply:
The label isn't currently required, but I think it might make sense for it to be since without a label a body kit alone would be more appropriate?

Reply:
I agree. It makes sense that a kit with the word "label" in it means that the prop is required.